README.md

```diff
@@ -9,11 +9,15 @@ This project shows how Fable can be used fullstack with a node.js server written
 - [Mono Framework](https://www.mono-project.com/download/stable/) for some tooling if working in non-Windows environment
 - An F# IDE, like Visual Studio Code with [Ionide extension](http://ionide.io/)
 
-## Installing dependencies
+## Installing & running (in dev mode)
 
-Type `npm install` to install dependencies (for both JS and F#) after cloning the repository or whenever dependencies change.
+- `.\.paket\paket.bootstrapper.exe`
+- `.\.paket\paket.exe update`
+- `dotnet restore`
```

Comment:
`dotnet restore` is being [called automatically after `npm install`](https://github.com/alfonsogarciacaro/fullstack-fable-example/blob/4d10b57b176034fcb8a3ce0f2bcf6eb2b5c6dafc/package.json#L4).